src/main/flight/imu.c

```diff
@@ -132,6 +133,11 @@ PG_RESET_TEMPLATE(imuConfig_t, imuConfig,
     .mag_declination = 0
 );
 
+const float (*getCraftRotationMaxtrix(void))[3]
+{
+    return (const float (*)[3])rMat;
```

Comment:
rMat is [3][3]. Byt passing it as float* is quite common in c (and new math PR mentioned above is more elegant)